### PR TITLE
Add a constant for the log in text

### LINKED ISSUE #3: Change log in text
As per Paul's [comment](https://dev.hmn.md/2018/08/27/docs-humanmade-com-login/comment-page-1/) we should change the [wording](https://github.com/humanmade/delegated-oauth2/blob/ab69b3f5555907a050e3066e1afc7bae745c1b48/inc/cookie/namespace.php#L21) from "Login with Delegated Auth" to something more user friendly such as "Log in with your HM Central account"

### PR DESCRIPTION
**Pull request by BronsonQuick**
Fixes #3.

### CODE REVIEW

File: inc/cookie/namespace.php [hunk 1]
```diff
@@ -18,7 +20,7 @@ function is_enabled() : bool {
  */
 function on_login_form() {
 	?>
-	<p><a href="<?php echo esc_url( get_authorize_url() ); ?>">Log In with Delegated Auth</a></p>
+	<p><a href="<?php echo esc_url( get_authorize_url() ); ?>"><?php echo esc_html( HM_DELEGATED_AUTH_LOGIN_TEXT ); ?></a></p>
```
Comment (peterwilsoncc) [inline inc/cookie/namespace.php]:
```php
// wp-config.php
define( 'HM_DELEGATED_AUTH_LOGIN_TEXT', false );
```

I'd probably do a stronger sanity check to ensure the defined value is valid, maybe `defined() && is_string()`, and fall back to the default if nothing will display.

**Review by peterwilsoncc — APPROVED** [review]
This will work.

I've noted that a stronger sanity check may be needed but it's up to you and whether you want to trust developers not to make a mistake.

File: inc/cookie/namespace.php (hunk 1 above)
Comment (BronsonQuick) [reply inc/cookie/namespace.php]: Good call. I've updated that now!

**Review by peterwilsoncc — APPROVED** [review]
Works for me.

**Pull request merged**